client/src/components/layout/ReviewTile.js

```diff
@@ -1,12 +1,57 @@
-import React from "react";
+import React, { useState, useRef, useEffect } from "react";
 
 const ReviewTile = (props) => {
+  const inputRef = useRef(null);
+  const [inputVisible, setInputVisible] = useState(false);
+  const [comments, setComments] = useState(props.review.comments);
+
+  const deleteReviewHandler = () => {
+    props.deleteReview(props.review);
+  };
+
+  function onClickOutSide(e) {
```

Comment:
Is there a reason this function is defined differently from your other functions in this file?